pkg/cluster/api/pdapi.go

```diff
@@ -465,8 +465,16 @@ func (pc *PDClient) DelPD(name string, retryOpt *clusterutil.RetryOption) error
 	endpoints := pc.getEndpoints(cmd)
 
 	_, err = tryURLs(endpoints, func(endpoint string) ([]byte, error) {
-		body, _, err := pc.httpClient.Delete(endpoint, nil)
-		return body, err
+		body, statusCode, err := pc.httpClient.Delete(endpoint, nil)
+		if err != nil {
+			if statusCode == 404 || bytes.Contains(body, []byte("not found, pd")) {
+				log.Debugf("PD node does not exist, ignore: %s", body)
```

Comment:
Better to print the error too.

Reply:
The error is just a wrap of body in this circumstance.